std/experimental/allocator/building_blocks/region.d

```diff
@@ -749,18 +741,18 @@ version(Posix) struct SbrkRegion(uint minAlign = platformAlignment)
     enum uint alignment = minAlign;
 
     /**
-    Rounds the given size to a multiple of the                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   `alignment`
+    Rounds the given size to a multiple of thew `alignment`
     */
-    size_t goodAllocSize(size_t n) shared
+    size_t goodAllocSize(size_t n) shared const pure nothrow @safe @nogc
     {
         return n.roundUpToMultipleOf(alignment);
     }
 
     /// Ditto
-    void[] allocate(size_t bytes) shared
+    void[] allocate(size_t bytes) shared nothrow @nogc
```

Comment:
This should also be marked as `@trusted` as it falls in line with how [Mallocator](https://github.com/dlang/phobos/blob/master/std/experimental/allocator/mallocator.d#L32) works.

Reply:
Done.